docs/docs/run_minder_server/installing_minder.md

```diff
@@ -0,0 +1,81 @@
+# Installing Minder
+
+## Keycloak Installation
+Minder is designed to operate without storing user credentials or personal information. To achieve this, it relies on an external identity provider. While Minder is compatible with any OpenID Connect (OIDC)-enabled identity provider, we have thoroughly tested it with Keycloak and thus recommend it for a seamless integration.
+
+### Getting Started with Keycloak
+To install Keycloak as your identity provider, please refer to the following resources for detailed instructions:
+
+- Keycloak Operator Installation Guide: [Keycloak Operator Installation](https://www.keycloak.org/operator/installation)
+- Keycloak Tutorials for Beginners: [Keycloak Tutorials](https://keycloak.ch/keycloak-tutorials/tutorial-1-installing-and-running-keycloak/)
+
+After the installation of Keycloak, there are specific settings and configurations required for Minder to function properly:
+
+1) **Realm Configuration:** Set up a dedicated realm in Keycloak for Minder's use.
+2) **Client Setup:** Create two separate clients within the realm:
+    - **minder-cli:** A client for command-line interactions.
+    - **minder-server:** A client for server-side operations.
+3) Identity Provider Linkage: Connect your chosen Identity Provider (e.g., GitHub, Google) to Keycloak. To facilitate this process, you may use the initialization script available at [Minder Identity Initialization Script](https://github.com/stacklok/minder/blob/main/identity/scripts/initialize.sh).
+4) OAuth2 Application: For GitHub integration, you will need to create a GitHub OAuth2 application to link user identities in Keycloak.
+
+## Postgres Installation
+Minder requires a dedicated Postgres database to store its operational data. The database must have a dedicated user with the necessary privileges and credentials.
+
+### Best Practices for Database Deployment
+It is recommended to use two distinct database users:
+
+- One for the Minder server operations.
+- Another solely for database migrations.
+
+You can find our database migration scripts at [Minder Database Migrations](https://github.com/stacklok/minder/tree/main/database/migrations).
+
+## Ingress Configuration
+Your ingress controller must be capable of handling both gRPC and HTTP/1 protocols. Please note that HTTP/2 compatibility has not been tested and is not guaranteed.
```

Comment:
gRPC is HTTP/2, so that should be sufficient.  Amazon's ALB apparently breaks some gRPC features when you just ask for HTTP/2 mode, though they've never documented what (seems related to streaming).